specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/textanalytics.json

```
@@ -747,6 +747,12 @@
       "type": "object",
       "description": "Supported parameters for a Healthcare task.",
       "properties": {
+        "fhirVersion": {
+          "type": "string",
+          "enum":[
+            "4.0.1"
```

IIUC, you mentioned "default" is also a valid option:
```suggestion
            "default"
```

---

after some discussion, we decided to keep default out because of potential confusion around what version the user is getting. Given that fhirVersion is very much a power user option, we want the user to know exactly what they are asking for/what to expect from the API so that they do not run into compatibility issues later